packages/react-radio/src/stories/RadioDescription.md

```diff
@@ -0,0 +1,3 @@
+Radio is a single option in a RadioGroup. It should not be used on its own: there should be at least two Radio items in a group.
+
+See `RadioGroup` for more details and examples.
```

Comment:
It's questionable whether it even makes sense to have stories for `Radio` that are separate from `RadioGroup`. Does anyone have opinions about whether to delete these Radio-only stories and just fold them into RadioGroup's stories?

Reply:
Potentially a user may want to manually group `Radio`s without a `RadioGroup` but I don't think we should encourage this (we provide `RadioGroup` for a reason). I'm in favor of dropping the `Radio` stories and adding them later if we see the need because we're getting lots of support requests.